### DISCUSSION TITLE
Alternative way to delete item (cell)

### DISCUSSION THREAD

**Discussion by cookie777** [DaiSugi01/Queens-game#57] [Q&A]
I found I can't use swipe cell kit which means I can't delete cell by swiping.
This is because because swipe cell kit  has strong competitive with diffable data source. (Cause error when deleting)

Can I just put delete button in the editing screen ?
(No feature of swiping cell now)

**Comment by DaiSugi01** [comment]
sure

**Comment by TakayasuNasu** [comment]
OK!
